src/appdev/ApplicationAPI.js

```diff
@@ -39,9 +39,9 @@ class ApplicationAPI {
     AppDevUtilities.tryCheckAppDevURL(this.getPath());
     const middleware = this.middleware();
     const routerGroups = this.routerGroups();
-    for (let i = 0; i < middleware.length; i++) {
-      this.express.use(middleware[i]);
-    }
+    middleware.forEach((m) => {
```

Comment:
Is `(m)` preferred over `m`? Can you check a style guide and see if that's preferred?

Reply:
[airbnb](https://github.com/airbnb/javascript) says always include parens

Reply:
ya i get an error without the parens